src/routes/+page.svelte

```diff
@@ -1,79 +1,83 @@
 <script lang="ts">
-  import "./page.scss";
-  import ContentfulRichText from "$lib/components/ContentfulRichText";
-  // import VideoCard from "$lib/components/VideoCard";
+  import "./(infoPages)/[topTierPage]/page.scss";
```

Comment:
Follow-up work: another point in favor of doing the `TODO` in that file (create a `<CardGroup />` wrapper component) now that we've got in a few different places

Reply:
![](https://media.tenor.com/hYBPeFN96qgAAAAC/brain-same-wavelength.gif)